test/LaunchDarkly.EventSource.Tests/EventSourceHttpBehaviorTest.cs

```diff
@@ -0,0 +1,193 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace LaunchDarkly.EventSource.Tests
+{
+    public class EventSourceHttpBehaviorTest : BaseTest
```

Comment:
These tests aren't new, they're just moved out of the former `EventSourceTests` and DRYd up a bit.